src/rust/inetstack/protocols/tcp/peer.rs

```diff
@@ -142,7 +146,6 @@ impl<const N: usize> SharedTcpPeer<N> {
         }
 
         // TODO: Check if we are binding to a non-local address.
-
```

Comment:
Could you rollback this change? The TODO comment does not concern the code block that follows.

Reply:
Done